docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.        chunksize = next_pow2(m)
</pre><p> </p><p>@NimaSarajpoor</p><p>Add the parameter <code>lookahead</code> to the function and pass the value <code>next_pow2(m)</code> as the argument. No need to re-compute it in every call of  <code>_forward_process</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done.</p>